scripts/tasks.sh

```diff
@@ -378,4 +381,65 @@ function set_kubeconfig_server() {
     fi
 }
 
+function taint_primaries() {
```

Comment:
does this work with other csi providers? if not should we add some note to docs

Reply:
No, OpenEBS and Longhorn do not tolerate the taint